src/features/constants.js

```diff
@@ -98,3 +98,30 @@ export const modalDeleteText = {
   title: 'Delete this class',
   body: 'This action will permanently delete this class and cannot be undone. Booked seat in this class will not be affected by this action.',
 };
+
+/**
+ * Custom styles for first option underlline in selector.
+ *@object
+ */
+export const styleFirstOption = {
```

Comment:
I didn't move it to the library because for the moment is not used in other part and is a little limited with the styles, but in the case pearson decide to used in other part, I will move to the library.